pkg/store/bucket.go

```diff
@@ -962,16 +958,18 @@ func (s *BucketStore) Series(req *storepb.SeriesRequest, srv storepb.Store_Serie
 			}
 
 			if err := srv.Send(storepb.NewSeriesResponse(&series)); err != nil {
```

Comment:
sorry, i was blind, i guess this just needs `if err = srv.Send...`

Reply:
otherwise the shadowed `err` will be an ineffectual assignment as CI complained :-)

Reply:
Wow! Thanks `go vet` :+1: